modules/space/shaders/star_ge.glsl

```diff
@@ -62,127 +56,164 @@ uniform dmat4 modelMatrix;
 const double PARSEC = 3.08567756E16;
 //const double PARSEC = 3.08567782E16;
 
-const vec2 corners[4] = vec2[4](
-    vec2(0.0, 0.0),
-    vec2(1.0, 0.0),
-    vec2(1.0, 1.0),
-    vec2(0.0, 1.0)
-);
-
-const float SunTemperature = 5800.0f;
-const float SunAbsMagnitude = 4.83f;
+// FRAGILE
+// All of these values have to be synchronized with the values in the optionproperty
+const int SizeCompositionOptionAppBrightness = 0;
+const int SizeCompositionOptionLumSize = 1;
+const int SizeCompositionOptionLumSizeAppBrightness = 2;
+const int SizeCompositionOptionLumSizeAbsMagnitude = 3;
+const int SizeCompositionOptionLumSizeAppMagnitude = 4;
+const int SizeCompositionOptionLumSizeDistanceModulus = 5;
```

Comment:
Damn, those variable names ;) But much clearer now than before! 👍

Reply:
I'd be happy if GLSL would have some enum construct instead ;)